elastic/apm-server
# package java-attacher alongside apm-server

## Motivation/summary

download and package the `java-attacher` when executing `make release`

## Checklist




## How to test these changes

run some different flavors of `mage package`, eg.
- `DEV=true SNAPSHOT=true PLATFORMS=linux/amd64 TYPES=docker mage package`
- `DEV=true SNAPSHOT=true PLATFORMS=linux/amd64 TYPES=tar.gz mage package`
- `DEV=true SNAPSHOT=true PLATFORMS=linux/amd64 TYPES=deb mage package`

and verify that `java-attacher.jar` is stored in the artifact alongside the `apm-server` binary

## Related issues

https://github.com/elastic/apm-server/issues/4824
